### DISCUSSION TITLE
Table of Contants

### DISCUSSION THREAD

**Discussion by 0xDTC** [TriliumNext/Trilium#3182] [General]
There is a bug in Table of contents

1.Numbering is not correct.
2.When you click on Table of contents Options it won't take you there.
3.There is no automatic underline separation in while adding header
![image](https://user-images.githubusercontent.com/95960398/194049072-ef2fa2e3-d1cd-4291-8889-5ca1d9aa1fa7.png)
![image](https://user-images.githubusercontent.com/95960398/194049185-4b24bd75-0718-49c1-acf6-2b076e894ece.png)
![image](https://user-images.githubusercontent.com/95960398/194049385-30c7ceba-ccaf-48dd-b46b-ae1107183510.png)
@nicollasricas

**Comment by zadam** [comment]
Hi, do you use a custom theme or the default built-in one?

Would it be possible to export the page into ZIP, so I can import it and debug? You can send it to me to zadam.apps@gmail.com

![image](https://user-images.githubusercontent.com/617641/194723648-4229c7a8-6005-4bc7-bff1-7f506205fec0.png)

Thanks!

**Reply by 0xDTC** [reply]
> Hi, do you use a custom theme or the default built-in one?
> Would it be possible to export the page into ZIP, so I can import it and debug? You can send it to me to

Here is the files 
[TABLE OF CONTENTS.zip](https://github.com/zadam/trilium/files/9741318/TABLE.OF.CONTENTS.zip)

And i would like to connect for some features if you don't mind

Thanks

**Reply by zadam** [reply]
Thanks, it was a small bug which caused the first digit of e.g. "10" not being shown, will be in the next release.

Feel free to create issues/discussions on the feature requests.
